spec-grpc/src/main/java/io/a2a/grpc/utils/JSONRPCUtils.java

```diff
@@ -509,7 +511,7 @@ public static String toJsonRPCResultResponse(Object requestId, com.google.protob
                 if (requestId instanceof String string) {
                     output.name("id").value(string);
                 } else if (requestId instanceof Number number) {
-                    output.name("id").value(number);
+                    output.name("id").value(number.intValue());
```

Comment:
I'm not sure that an `int` is correct here? What if the id is bigger than `Integer.MAX`?
Should we use instead a long?